contracts/script/deploy/AlignedLayerDeployer.s.sol

```diff
@@ -109,6 +109,8 @@ contract AlignedLayerDeployer is ExistingDeploymentParser {
         //deploy service manager router
         serviceManagerRouter = new ServiceManagerRouter();
 
+        emptyContract = new EmptyContract();
```

Comment:
I'm not sure why I needed this for the deployer to work since we didn't need it before

Reply:
removed